channeldb/graph.go

```diff
@@ -4,6 +4,7 @@ import (
 	"bytes"
 	"crypto/sha256"
 	"encoding/binary"
+	"encoding/hex"
```

Comment:
Commit body should give a high level overview of the new approach.

Reply:
Added body and split up final commit in multiple smaller changes.